sas/static/sas/js/picture.js

```diff
@@ -0,0 +1,191 @@
+/**
+ * @typedef PictureIdentification
+ * @property {number} id The actual id of the identification
+ * @property {UserProfile} user The identified user
+ */
+
+
+document.addEventListener("alpine:init", () => {
+  Alpine.data("picture_viewer", () => ({
+    /**
+     * All the pictures that can be displayed on this picture viewer
+     * @type Picture[]
+     **/
+    pictures: [],
+    /**
+     * The users identified on the currently displayed picture
+     * @type PictureIdentification[]
+     **/
+    identifications: [],
+    /**
+     * The currently displayed picture
+     * @type Picture
+     **/
+    current_picture: undefined,
```

Comment:
Pour éviter pas mal des erreurs qui sont pendant le chargement
Mais faut quand même changer un peu le template
```suggestion
    current_picture: {
        id: null,
        name: gettext("loading"),
        display_name: gettext("loading"),
        compressed_url: window.location.href,
        profile_url: window.location.href,
        full_size_url: window.location.href,
        date: new Date(),
    },
```